torch/_tensor_docs.py

```diff
@@ -1457,11 +1457,11 @@ def callable(a, b) -> number
                r"""
 narrow_copy(dimension, start, length) -> Tensor
 
-Same as :meth:`Tensor.narrow` except returning a copy rather
-than shared storage.  This is primarily for sparse tensors, which
-do not have a shared-storage narrow method.  Calling ```narrow_copy``
-with ```dimemsion > self._sparseDims()``` will return a copy with the
-relevant dense dimension narrowed, and ```self.shape``` updated accordingly.
+Same as :func:`torch.narrow` except that :meth:`~FloatTensor.narrow_copy` is a Tensor method
```

Comment:
Does `~torch.Tensor.narrow_copy` not work?

Reply:
just tried, but no, it doesn't work